bash/cloner.sh

```diff
@@ -25,26 +38,54 @@ make_folder () {
   echo
 }
 
-clone_repos () {
-  curl -s https://api.github.com/users/$account_name/repos | jq -c '.[]' |
+check () {
+  if [ -n "$1" ]
+    then # non-null/non-zero string check ... aka exists!
+      echo "success"
+      return 0; # return something similar to an exit code.
+    else
+      # echo "error"
+      return 1; # return something similar to an exit code.
+  fi
+}
 
-  while read i;
-  do
-    val=$(jq -r '.clone_url' <<< "$i");
-    (git clone -q $val) &> /dev/null & 
-  done;
+get_repos_by_page () {
+  local page=$1;
```

Comment:
The below failure scenarios need to be addressed in a future pr.

Reply:
starting to be addressed in https://github.com/murjax/Github-Repo-Cloner/pull/45